profiles/centos-stream/repos/centos-stream-10.repo

```diff
@@ -0,0 +1,273 @@
+###### BASE OS ###########
+[baseos]
+name=CentOS Stream 10 - BaseOS
+baseurl=https://composes.stream.centos.org/stream-10/production/latest-CentOS-Stream/compose/BaseOS/x86_64/os/
```

Comment:
centos-stream-9.repo uses `$basearch` instead of a hardcoded architecture, for this repo and the rest, so we should probably do the same here.

```suggestion
baseurl=https://composes.stream.centos.org/stream-10/production/latest-CentOS-Stream/compose/BaseOS/$basearch/os/
```